src/lib/pmd/PmdSupport.ts

```diff
@@ -34,26 +28,9 @@ export abstract class PmdSupport extends AsyncCreatable {
 		return classpathEntries;
 	}
 
-	/**
-	 * Accepts a child process created by child_process.spawn(), and a Promise's resolve and reject functions.
-	 * Resolves/rejects the Promise once the child process finishes.
-	 * @param cp
-	 * @param res
-	 * @param rej
-	 */
-	protected abstract monitorChildProcess(cp: ChildProcessWithoutNullStreams, res: (string) => void, rej: (string) => void): void;
 
 	protected abstract buildCommandArray(): Promise<[string, string[]]>;
```

Comment:
Does this method still need to be defined here if it's defined in the parent class?

Reply:
@jfeingold35 I don't believe it's defined in any parent class. It's defined in PmdWrapper which is a child of PmdSupport.